src/pages/u/[handle].tsx

```diff
@@ -4,14 +4,51 @@ import withAccessType from '@/lib/hoc/withAccessType';
 import { CookieService, PermissionService } from '@/lib/services';
 import type { PublicProfile } from '@/lib/types/apiResponses';
 import { CookieType } from '@/lib/types/enums';
+import defaultProfilePicture from '@/public/assets/profile-pics/adorable0.png';
+import styles from '@/styles/pages/u.module.scss';
+import Image from 'next/image';
 import type { GetServerSideProps } from 'next/types';
 
 interface UserProfilePageProps {
   user: PublicProfile;
 }
 
 const UserProfilePage = ({ user }: UserProfilePageProps) => {
-  return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  // return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  return (
+    <div className={styles.profilePage}>
+      <div className={styles.cardWrapper}>
+        <div className={styles.banner} />
+        <div className={styles.profileCard}>
+          <div className={styles.cardHalf}>
+            <div className={styles.horizontal}>
+              <Image
+                src={user.profilePicture ?? defaultProfilePicture}
```

Comment:
Also, doesn't look like this is working with an uploaded pfp for some reason. maybe use the acm testing account as an example with a pfp and your personal acct for an example with no pfp

![Screenshot 2023-05-08 at 10 51 02 AM](https://user-images.githubusercontent.com/33165426/236895137-78fc0f8e-24df-49c4-b854-6c6489908bc6.jpg)

Reply:
fixed in Clean up code 1